manage

```diff
@@ -272,7 +271,7 @@ Or on Discord: \e[94m\e[4mhttps://discord.gg/RXSTvaUvuu\e[0m" | tee -a "$logfile
     fi
     
     format_logfile "$logfile" #remove escape sequences from logfile
-    mv "$logfile" "$(echo "$logfile" | sed 's+-incomplete-+-fail-+g')"
+    mv "$logfile" "$(echo "$logfile" | sed 's+-incomplete-+-fail-+g')" && logfile="$(echo "$logfile" | sed 's+-incomplete-+-fail-+g')"
```

Comment:
What effect does changing the value of the `$logfile` variable do? The `manage` script exits shortly afterwards.

Reply:
The logfile still needed by `log_diagnose`.

Reply:
![image](https://user-images.githubusercontent.com/88134003/139770561-2b66a627-86c3-467b-9453-5c04265fdbd4.png)

Reply:
Looks like we're dealing with duplicate code. There's no need to run `log_diagnose` twice in `manage install`.